File: Iniciante/URI 1006.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
    
var a = parseFloat(lines.shift());
var b = parseFloat(lines.shift());
var c = parseFloat(lines.shift());
var media = ((a * 2) + (b * 3) + (c * 5)) / 10;
   
console.log('MEDIA = ' + media.toFixed(1));

File: Iniciante/URI 1006.r
input = file('stdin', 'r')

a = as.double(readLines(input, n=1))
b = as.double(readLines(input, n=1))
c = as.double(readLines(input, n=1))
media = (a * 2 + (b * 3 + c * 5)) / 10

write(sprintf("MEDIA = %.1f", media), '')